engine/shaderc/src/shaderc.h

```diff
@@ -135,7 +135,6 @@ namespace dmShaderc
         ShaderCompilerOptions()
         : m_Version(330)
         , m_EntryPoint("main")
-        , m_Stage(SHADER_STAGE_VERTEX)
```

Comment:
This is removed from the options. This will probably require a new release of PS4/PS5 extensions for next release!